### PR TITLE
Qai 181 etl timescale setup

### PR DESCRIPTION
**Pull request by twz00**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ gradle/*
 # files
 gradlew
 gradlew.bat
+.env
```
Comment (Ivsucram) [inline .gitignore:12]: According to your git commit messages, you have deleted the env file. Therefore, remove this line from your gitignore as well.

File: build.gradle [hunk 2]
```diff
@@ -45,6 +45,9 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-webflux")
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
     implementation("org.jetbrains.kotlin:kotlin-reflect")
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    implementation 'org.springframework.boot:spring-boot-starter-jdbc'
```
Comment (Ivsucram) [inline build.gradle]: Edit the line to follow the convention of the rest of the file, i.e., parenthesis and double quotes for string.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -2,6 +2,17 @@
 spring.application.name = quant-ai-persistence-etl
 server.port = 10070
 
+# Timescale
+spring.datasource.url = jdbc:postgresql://localhost:10003/quant-ai 
+# spring.datasource.url = jdbc:postgresql://quant-ai-timescale-db:5432/quant-ai
```
Comment (Ivsucram) [inline src/main/resources/application.properties]: Remove commented line

**Review by Ivsucram — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 3 above)
Comment (twz00) [reply src/main/resources/application.properties]: Removed

File: .gitignore (hunk 1 above)
Comment (twz00) [reply .gitignore:12]: I will still need the .env file which contains my Cryptocompare API keys 
So I will leave it in the gitignore file

File: build.gradle (hunk 2 above)
Comment (twz00) [reply build.gradle]: Changes made to follow the convention

**Pull request merged**